src/components/AddItem/AddItem.test.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import App from '../../App';
+
+it('should add a new item to list', async () => {
+  render(<App />);
+  const input = await screen.findAllByRole('textbox');
+  userEvent.type(input[0], 'beans');
+
+  expect(input[0]).toBeInTheDocument();
```

Comment:
This isn't exactly a behavior test. You're just checking if the input element is within the document, but on the Add page it will be there since that is the purpose of that page. You want to test to make sure that after you type into the field and submit the page, that the new item is added to the displayed list.